package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/aio-lib-console",
-  "version": "5.2.0",
+  "version": "5.2.1",
```

Comment:
Revert - this will get updated automatically when we release

Reply:
Done, reverted this. Also added tests and a bug fix.